setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
 ignore =
-    E501,F403,F405,E226,E265,E731,E402,E266,
+    E501,F403,F405,E226,E265,E731,E402,E266,F999,
```

Comment:
What fails without F999? On my machine flake8 is still happy without this option.